package.json

```diff
@@ -113,7 +111,9 @@
     "webpack": "^2.6.1"
   },
   "dependencies": {
-    "prop-types": "^15.5.10"
+    "babel-runtime": "^6.23.0",
```

Comment:
Do you know the impact this is having to the final bundle size? I'd rather remove the stuff not supported by IE11.